client/src/components/graph.jsx

```diff
@@ -0,0 +1,207 @@
+import { Component, Fragment } from 'preact'
+import { getGraph } from '../api/scoreboard'
+import withStyles from './jss'
+
+const width = window.innerWidth
+const height = 400
+const stroke = 5
+const axis = 20
+const axisGap = 10
+const day = 24 * 60 * 60 * 1000
+const viewBox = `${-stroke - axis} ${-stroke} ${width + stroke * 2 + axis} ${height + stroke * 2 + axis + axisGap}`
```

Comment:
`width` and thus `viewBox` should be calculated within the component so we can handle window resizes (recalculate & redraw on the resize event)